engine-trans.ts

```diff
@@ -0,0 +1,172 @@
+import console from 'console';
```

Comment:
what is this file? why is it in the PR?

Reply:
this is the code mode code that got committed. can be removed